src/StatusAlert/StatusAlert.test.jsx

```diff
@@ -24,65 +35,78 @@ describe('<StatusAlert />', () => {
+      expect(wrapper.find('withDeprecatedProps(Button)')).toHaveLength(1);
     });
 
     it('renders non-dismissible view', () => {
       wrapper = mount(<StatusAlert {...defaultProps} dismissible={false} />);
       const statusAlertDialog = wrapper.find('.alert-dialog');
 
       expect(statusAlertDialog.text()).toEqual(dialog);
-      expect(wrapper.find('button')).toHaveLength(0);
+      expect(wrapper.find('withDeprecatedProps(Button)')).toHaveLength(0);
     });
 
     it('renders custom aria-label view', () => {
       const customLabel = 'Dismiss this alert post-haste!';
       wrapper = mount(<StatusAlert {...defaultProps} closeButtonAriaLabel={customLabel} />);
-      const button = wrapper.find('button').at(0);
-
+      const button = wrapper.find('withDeprecatedProps(Button)').at(0);
       expect(button.prop('aria-label')).toEqual(customLabel);
     });
   });
 
+  const assertStatusAlertOpen = (isOpen) => {
+    expect(wrapper.find('.alert').hasClass('show')).toEqual(isOpen);
+  };
   describe('props received correctly', () => {
     it('component receives props', () => {
-      wrapper = mount(<StatusAlert dialog={dialog} onClose={() => {}} />);
-
-      statusAlertOpen(false, wrapper);
+      wrapper = shallow(
+        <StatusAlert theme={theme} dialog={dialog} onClose={() => {}} />,
+      );
+      assertStatusAlertOpen(false);
       wrapper.setProps({ open: true });
-      statusAlertOpen(true, wrapper);
+      assertStatusAlertOpen(true);
     });
 
     it('component receives props and ignores prop change', () => {
-      wrapper = mount(<StatusAlert {...defaultProps} />);
+      wrapper = shallow(<StatusAlert {...defaultProps} />);
 
-      statusAlertOpen(true, wrapper);
+      assertStatusAlertOpen(true);
       wrapper.setProps({ dialog: 'Changed alert dialog' });
-      statusAlertOpen(true, wrapper);
+      assertStatusAlertOpen(true);
     });
   });
 
   describe('close functions properly', () => {
     beforeEach(() => {
-      wrapper = mount(<StatusAlert {...defaultProps} />);
+      wrapper = shallow(<StatusAlert {...defaultProps} />);
+      event = { preventDefault: jest.fn() };
     });
 
     it('closes when x button pressed', () => {
-      statusAlertOpen(true, wrapper);
-      wrapper.find('button').at(0).simulate('click');
-      statusAlertOpen(false, wrapper);
+      assertStatusAlertOpen(true);
+      wrapper.find('withDeprecatedProps(Button)').at(0).simulate(
+        'click',
+        event,
+      );
+      assertStatusAlertOpen(false);
     });
 
     it('closes when Enter key pressed', () => {
-      statusAlertOpen(true, wrapper);
-      wrapper.find('button').at(0).simulate('keyDown', { key: 'Enter' });
-      statusAlertOpen(false, wrapper);
+      assertStatusAlertOpen(true);
+      wrapper.find('withDeprecatedProps(Button)').at(0).simulate(
+        'keyDown',
+        { ...event, key: 'Enter' },
+      );
+      assertStatusAlertOpen(false);
```

Comment:
Changes in this file seem unrelated to the purpose of this PR. Is there a reason this is all needed?